src/services/codefixes/helpers.ts

```diff
@@ -219,7 +219,8 @@ export function addNewNodeForMemberSymbol(
     switch (kind) {
         case SyntaxKind.PropertySignature:
         case SyntaxKind.PropertyDeclaration:
-            const flags = quotePreference === QuotePreference.Single ? NodeBuilderFlags.UseSingleQuotesForStringLiteralType : undefined;
+            let flags = NodeBuilderFlags.NoTruncation;
+            flags |= quotePreference === QuotePreference.Single ? NodeBuilderFlags.UseSingleQuotesForStringLiteralType : 0;
             let typeNode = checker.typeToTypeNode(type, enclosingDeclaration, flags, getNoopSymbolTrackerWithResolver(context));

```

Comment:
Searching for `checker.typeToTypeNode`, it looks like there are a few other calls that are missing that flag. Some shouldn't have it, but I see quite a few that definitely should. May be good to audit all instances of that call in `services`.